src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -79,6 +79,7 @@ def serialize(self, obj, attrs, user):
             "triggers": attrs.get("triggers", []),
             "projects": sorted(attrs.get("projects", [])),
             "includeAllProjects": obj.include_all_projects,
+            "owner": obj.owner().get_actor_id(),
```

Comment:
How will this be used by the frontend?

Reply:
I think they have a wrapper for this. Not 100% sure how they handle it, but we do the same thing for GroupOwner.